mobile/src/common/types/permission/permission-item.type.ts

```diff
@@ -0,0 +1 @@
+export { type PermissionsGetAllItemResponseDto as PermissionItem } from 'guruhub-shared/common/types/permission/permission';
```

Comment:
Remove this file

Reply:
I am using this type in mobile/src/navigation/app/helpers/get-allowed-screens.helper.ts

Reply:
oh, I see, then don't rename it and use it as it is. 
```
// common/types/permissions/permissions.ts
export { type PermissionsGetAllItemResponseDto } from 'guruhub-shared/common/types/types'
```

Reply:
Ok, I see, then don't rename it, just export as it is
```
// common/types/permissions/permissions

export { type PermissionsGetAllItemResponseDto } from 'guruhub-shared/common/types/types';
```